x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/action/CreateEnrollmentTokenTests.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.core.security.action;
+
+import org.elasticsearch.common.io.stream.BytesStreamOutput;
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.test.ESTestCase;
+import org.elasticsearch.xpack.core.security.action.enrollment.CreateEnrollmentTokenResponse;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+
+public class CreateEnrollmentTokenTests extends ESTestCase {
```

Comment:
```suggestion
public class CreateEnrollmentTokenResponseTests extends ESTestCase {
```

Reply:
You can extend `AbstractXContentTestCase` which was created for this kind of tests